curling/command.py

```diff
@@ -80,9 +80,14 @@ def new(config, lib_api=None):
         return
 
     method = getattr(api, config.request.lower())
+    kwargs = {
+        'data': config.data
+    }
+    query_dict = OrderedDict(urlparse.parse_qsl(str(url.query)))
+    kwargs.update(query_dict)
 
     try:
-        res = method(data=config.data)
+        res = method(**kwargs)
```

Comment:
So, I fixed it in the command, but because of the signature the methods in the lib (`def <name>(self, data, headers=None, **kwargs)`) this is still an issue. This is an issue that is already present in slumber, too - to fix it, the query dict would need to be passed as a separate parameter instead of accepting keyword arguments like this.

TL;DR: not a regression.